link_shorterner/test/shortener_shortener_SUITE.erl

```diff
@@ -0,0 +1,70 @@
+-module(shortener_shortener_SUITE).
```

Comment:
I suggest `shortly` as the name of the app, instead of shortener

Reply:
+1